src/components/Table/Table.jsx

```diff
@@ -357,6 +357,8 @@ const Table = props => {
       options.hasSearch ||
       options.hasRowActions ||
       options.hasRowCountInHeader ||
+      options.hasColumnSelection ||
+      actions.toolbar.onDownloadCSV ||
```

Comment:
do we need to conditionally check actions and toolbar for null?  Can actions or toolbar ever be null?
actions?.toolbar?.onDownloadCSV

Reply:
Good catch. `options` can also be null